igvf_dump_metadata.py

```diff
@@ -32,17 +32,20 @@
 # set the properties/columns here:
 # any column with all rows = None will be dropped in final tables
 # can add 'summary' to basic_props when it's available for all objs
-basic_props = ['@id', 'accession', 'aliases', 'status', 'audit']
+basic_props = ['@id', 'aliases', 'status', 'audit']
 # removed modifications, treatments, donors linkTo objs (documents, sources), summary in output_props is probably enough?
 # is 'documents' useful?
-# for the file-set 'paired' with the input file-set (could be either measurement set or auxiliary set), 
-# not doing a secondary query for their linkTo obj for now, just output their accessions
+# for the file-set 'paired' with the input file-set (could be either measurement set or auxiliary set),
+# skip a the query for their link sample, just output their ids
 link_obj_props = {'input_file_sets': ['measurement_sets', 'auxiliary_sets', 'control_for', 'control_file_sets', 'samples', 'library_construction_platform', 'assay_term', 'documents'],
                   'samples': ['sample_terms', 'biomarkers', 'modifications', 'sorted_from', 'part_of', 'donors', 'construct_library_sets', 'treatments', 'originated_from', 'sources', 'multiplexed_samples', 'demultiplexed_from', 'barcode_sample_map', 'targeted_sample_term', 'cell_fate_change_treatments', 'cell_fate_change_protocol'],
-                  'files': ['derived_from']
+                  'files': ['derived_from'],
+                  'auxiliary_sets': ['library_construction_platform'],
+                  'measurement_sets': ['assay_term', 'library_construction_platform']
```

Comment:
could you add publication_identifiers for input file sets & samples. We wouldn't have an audit for this so there's no way to otherwise catch that they all share the same one.